oaknational/Oak-Web-Application
# Feat/connect to versioned search api - LESQ-581

## Description

Music year: 2013

- Removes construct elastic query (We now construct in the search api)
- Adds version to endpoint in browser config


## How to test

1. Go to https://deploy-preview-2223--oak-web-application.netlify.thenational.academy
search should work the same as before




## Acceptance Criteria:

- [x]  Release search api version to staging
- [x]  Delete construct query from OWA and just send query object
- [x]  Update staging config with search endpoint with version number (2)
- [x]  Test search is working
- [x]  Release search api version to production
- [x]  Update production config with search endpoint with version number (2)
- [x]  Test search is working
